src/open_source_python_template/db/db.py

```diff
@@ -0,0 +1,98 @@
+from psycopg2 import pool
+from datetime import datetime, timezone
+
+
+class DBConfig:
+    DATABASE_HOST = ""
+    DATABASE_NAME = "task-db"
+    DATABASE_USER = "postgres"
+    DATABASE_PASSWORD = ""
```

Comment:
It is kind of insecure for host and password to be hardcoded directly into the codebase.